test/support/application_fixture_helper.rb

```diff
@@ -1,7 +1,10 @@
-# typed: false
+# typed: true
 # frozen_string_literal: true
 
 module ApplicationFixtureHelper
+  # So sorbet knows self.raise exists
+  include Kernel
```

Comment:
`requires_ancestor` is the right way to fix this.

Reply:
Unfortunately it doesn't -- going to use `T.unsafe` for now

Reply:
Ah, it is because `requires_ancestor` is not enable in our sorbet config. Let's add `--enable-experimental-requires-ancestor` to sorbet/config

Reply:
Ah perfect, adding!